src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Component,
+  Input,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyLink
+} from '../link-list/types/link';
+
+import {
+  NeedsAttention
+} from '../needs-attention/types/needs-attention';
+
+import {
+  Configuration
+} from './types/configuration';
+
+@Component({
+  selector: 'sky-action-hub-buttons',
+  template: `<ng-content></ng-content>`
+})
+export class SkyActionHubButtonsComponent {}
```

Comment:
We can discuss in flex time if you'd like but historically we have placed even these simple components in their own file (actually typically even with a separate `html` file) so that they are easier to find if needed later.